chapters/synchronous.tex

```diff
@@ -1043,6 +1044,29 @@ \subsection{Sub-Clock Partitioning}\label{sub-clock-partitioning}
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Forbidding systems of equations involving different sub-clocks mean that the following is forbidden:
+\begin{lstlisting}[language=modelica]
+  Real x=sample(time, Clock(1,100));
+  Real y;
+  Real z=subSample(y, 2)+x;
+equation
+  z=0;
+\end{lstlisting}
+Here \lstinline!x! and \lstinline!z! are part of one sub-clock partition and \lstinline!y! of another.
+The variables \lstinline!y! and \lstinline!z! are part of a system of equations spanning two sub-clocks, that is not legal.
```

Comment:
I'd find this more readable (with _allowed_ being the natural opposite of _forbidden_):
```suggestion
The variables \lstinline!y! and \lstinline!z! are part of a system of equations spanning two sub-clocks, which is not allowed.
```